src/WebJobs.Script/Host/ScriptHost.cs

```diff
@@ -746,9 +746,6 @@ private void InitializeWorkers()
                 // TODO: We still have some hard coded languages, so we need to handle them. Remove this switch once we've moved away from that.
                 switch (_language.ToLowerInvariant())
                 {
-                    case LanguageWorkerConstants.JavaLanguageWorkerName:
```

Comment:
Can't this switch and TODO go away now and be a simple if(!DotNetLanguageWorker) check?